mmtf-api/src/main/java/org/rcsb/mmtf/api/DataApiInterface.java

```diff
@@ -144,7 +144,7 @@
 
 
 	/**
-	 * Returns the a list length N models, indicating the number of (internal) chains in each model.
+	 * Returns an array of length N models, indicating the number of (internal) chains in each model.
```

Comment:
Yes - good point! Thanks for this Jose!

On Fri, Mar 25, 2016 at 4:28 PM Jose Manuel Duarte <notifications@github.com>
wrote:

> In mmtf-api/src/main/java/org/rcsb/mmtf/api/DataApiInterface.java
> <https://github.com/rcsb/mmtf-java/pull/3#discussion_r57502207>:
>
> > @@ -144,7 +144,7 @@
> >
> >
> >  	/**
> > -	 * Returns the a list length N models, indicating the number of (internal) chains in each model.
> > +	 * Returns an array of length N models, indicating the number of (internal) chains in each model.
>
> We need to explain well somewhere (perhaps on the class docs on top of
> this file) what we mean by internal chains and be consistent with the
> nomenclature throughout
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rcsb/mmtf-java/pull/3/files/a839340374ccc73595e675af7ed33e1208b13213#r57502207>
>